crates/sc-subspace-chain-specs/src/lib.rs

```diff
@@ -16,27 +16,32 @@
 
 //! Chain specification data structures tailored for Subspace.
 
-mod utils;
+pub mod utils;
```

Comment:
making `utils` public does not look great, can we directly make the APIs needed to be public instead of publicing the whole unils module?